packages/types/src/pricing/service.ts

```diff
@@ -157,11 +161,41 @@ export interface IPricingModuleService {
     sharedContext?: Context
   ): Promise<RuleTypeDTO[]>
 
-  deleteRuleTypes(
-    ruleTypeIds: string[],
+  deleteRuleTypes(ruleTypeIds: string[], sharedContext?: Context): Promise<void>
+
+  retrievePriceSetMoneyAmountRules(
+    id: string,
+    config?: FindConfig<PriceSetMoneyAmountRulesDTO>,
+    sharedContext?: Context
+  ): Promise<PriceSetMoneyAmountRulesDTO>
+
+  listPriceSetMoneyAmountRules(
+    filters?: FilterablePriceSetMoneyAmountRulesProps,
+    config?: FindConfig<PriceSetMoneyAmountRulesDTO>,
+    sharedContext?: Context
+  ): Promise<PriceSetMoneyAmountRulesDTO[]>
+
+  listAndCountPriceSetMoneyAmountRules(
+    filters?: FilterablePriceSetMoneyAmountRulesProps,
+    config?: FindConfig<PriceSetMoneyAmountRulesDTO>,
+    sharedContext?: Context
+  ): Promise<[PriceSetMoneyAmountRulesDTO[], number]>
+
+  createPriceSetMoneyAmountRules(
+    data: CreatePriceSetMoneyAmountRulesDTO[],
+    sharedContext?: Context
+  ): Promise<PriceSetMoneyAmountRulesDTO[]>
+
+  updatePriceSetMoneyAmountRules(
+    data: UpdatePriceSetMoneyAmountRulesDTO[],
+    sharedContext?: Context
+  ): Promise<PriceSetMoneyAmountRulesDTO[]>
+
+  deletePriceSetMoneyAmountRules(
+    ids: string[],
```

Comment:
question: will you include the overload in this pr?

Reply:
Not yet, it'll come in a follow up pr.